src/components/Settings/Settings.react.js

```diff
@@ -685,7 +687,124 @@ class Settings extends Component {
     });
   };
 
+  handleAvatarImageChange = e => {
+    e.preventDefault();
+    // eslint-disable-next-line no-undef
+    let reader = new FileReader();
+    let file = e.target.files[0];
+    reader.onloadend = () => {
+      this.setState({
+        file: file,
+        imagePreviewUrl: reader.result,
+        isAvatarAdded: true,
+      });
+    };
+    reader.readAsDataURL(file);
+  };
+
+  removeAvatarImage = () => {
+    this.setState({
+      file: '',
+      isAvatarAdded: false,
+      imagePreviewUrl: '',
```

Comment:
Reset `avatarSrc` while removing Avatar Image, this may be the reason for the bug.

Reply:
Tried it, not working. Will look into this today, appreciate your help!

Reply:
Resolved the error, turns out the input field was not reseting the value of target on uploading the same image.